src/main/scala/io/evvo/agent/agent.scala

```diff
@@ -57,56 +57,56 @@ abstract class AAgent[Sol](
             step()
           } catch {
             case e: Exception => {
-              log.error(
-                f"${this}: Agent ${name} encountered an exception during a step, " +
+              log.warn(
+                f"${this}: ${name} encountered an exception during a step, " +
                   f"stack trace: ${e.getStackTrace.mkString("\n")}"
               )
             }
           }
           Thread.sleep(waitTime.toMillis)
 
           // this is arbitrary
-          if (numInvocations % 33 == 0) {
-            val nextInformation = population.getInformation()
-            waitTime = strategy.waitTime(nextInformation)
+//          if (numInvocations % 1000 == 0) {
```

Comment:
We should prob just delete this part, I don't think we'll want that log again, it seems pretty specific